tests/spark/test_text_operations.py

```diff
@@ -279,19 +309,23 @@ def test_n_gram_operations_success():
             model = pipeline.fit({1})
             {2} = model.transform({1})
             """.format(params[GenerateNGramsOperation.N_PARAM],
-                    inputs, outputs,
+                    inputs[0], outputs[0],
                     json.dumps(zip(params[GenerateNGramsOperation.ATTRIBUTES_PARAM],
                                     params[GenerateNGramsOperation.ALIAS_PARAM]))))
 
     result, msg = compare_ast(ast.parse(code), ast.parse(expected_code))
-    assert result, msg
-
-    # Test NGramOperations
+    assert result, msg + debug_ast(code, expected_code)
 
 
+# Test NGramOperations
 def test_n_gram_operations_failure():
```

Comment:
The name should include a resume about the cause of the failure, something like "test_n_gram_missing_params_failure"